serverless/knative_serving/serverless-applications.adoc

```diff
@@ -35,6 +35,9 @@ include::modules/kn-service-describe.adoc[leveloffset=+1]
 include::modules/verifying-serverless-app-deployment.adoc[leveloffset=+1]
 include::modules/interacting-serverless-apps-http2-gRPC.adoc[leveloffset=+1]
 
+// Using Knative services w/ restrictive NetworkPolicies
+include::modules/serverless-services-network-policies.adoc[leveloffsset=+1]
```

Comment:
typo (leveloffsset -> leveloffset), causing the heading to show up at the wrong level